apps/design/frontend/src/ballots_screen.tsx

```diff
@@ -142,104 +171,180 @@ function BallotStylesTab(): JSX.Element | null {
           created districts, precincts, and contests.
         </P>
       ) : (
-        <Table style={{ maxWidth: '40rem' }}>
-          <thead>
-            <tr>
-              <TH>Precinct</TH>
-              <TH>Ballot Style</TH>
-              {election.type === 'primary' && <TH>Party</TH>}
-              <TH />
-            </tr>
-          </thead>
-          <tbody>
-            {precincts.flatMap((precinct) => {
-              if (!hasSplits(precinct)) {
-                const precinctBallotStyles = ballotStyles.filter(
-                  (ballotStyle) =>
-                    ballotStyle.precinctsOrSplits.some(
-                      ({ precinctId, splitId }) =>
-                        precinctId === precinct.id && splitId === undefined
-                    )
-                );
-                return precinctBallotStyles.map((ballotStyle) => (
-                  <tr key={precinct.id + ballotStyle.id}>
-                    <TD>{precinct.name}</TD>
-                    <TD>{ballotStyle.id}</TD>
-                    {election.type === 'primary' && (
-                      <TD>
+        <Column style={{ gap: '1rem', maxWidth: '40rem' }}>
+          <FinalizeBallotsCallout>
+            <Row
+              style={{ justifyContent: 'space-between', alignItems: 'center' }}
+            >
+              <Row style={{ gap: '0.5rem' }}>
+                {ballotsFinalizedAt ? (
+                  <React.Fragment>
+                    <Icons.Done color="primary" />
+                    <H3>Ballots are Finalized</H3>
+                  </React.Fragment>
+                ) : (
+                  <React.Fragment>
+                    <Icons.Info />
+                    <div>
+                      <H3>Ballots are Not Finalized</H3>
+                      <div style={{ marginTop: '0.5rem' }}>
+                        Proof each ballot style, then finalize ballots.
+                      </div>
+                    </div>
+                  </React.Fragment>
+                )}
+              </Row>
+              <Button
+                icon="Done"
+                color="primary"
+                fill="outlined"
+                disabled={
+                  ballotsFinalizedAt !== null ||
+                  setIsBallotProofingCompleteMutation.isLoading
+                }
+                onPress={() => setIsConfirmingFinalize(true)}
+              >
+                Finalize Ballots
+              </Button>
```

Comment:
I think this structure is a little hard to reason about because (1) it has nested rows and (2) I'd expect a `Row` to take the full width of its container, so I'm surprised the `Button` appears to the right of the text contained by the `Row`. Is there a strong reason to use this structure over `flex` styling?

Reply:
All `Row` does is apply `display: flex; flex-direction: row`. Are you suggesting just applying those styles inline instead?